Frends.SMTP.SendEmail/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.3.0] - 2024-09-30
+### Added
+- [Breaking] Added parameter AcceptAllCerts which allows bypassing SSL/TLS certificate validation for SMTP servers.
```

Comment:
If this is a breaking change, then we should bump version to 2.0.0. If it is however not breaking and the defaults still persist the same functionality as before, then we can remove the [Breaking] text here.